content/en/contribute/code/repository-checklist.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Repository Checklist"
+linkTitle: "Repository Checklist"
+weight: 5
+description: >
+  Checklist to consider when creating repositories under Medic's GitHub account
+---
+
+## Repository Checklist
+When creating a new repository under [Medic's GitHub organization](https://github.com/medic), the contributor(s) should at least do the following:
+
+### Source Control
+- [ ] The `main` branch is locked via [branch protection rules](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/managing-a-branch-protection-rule).
+- [ ] Merges are done through PRs.
+- [ ] Issue templates exist.
+- [ ] PR template exists.
+- [ ] PRs reference related issues.
+- [ ] Commit formats follow the [guidelines]({{< ref "workflow/#commits" >}}). 
+- [ ] Secrets are not part of the commit history or made public.
+- [ ] For public repositories the `main` branch should contain the following files:
+    - `LICENSE` ([example](https://github.com/medic/cht-core/blob/master/LICENSE))
+    - `README.md`
+
+### Testing
+- [ ] Unit tests cover the majority of the code (>90% if possible).
+- [ ] If applicable, integration tests run to test the solution e2e.
+
+### CI/CD
+- [ ] Repo runs GitHub Actions CI with automated build and test on each PR.
+- [ ] `Main` branch is always shippable.
+- [ ] If applicable, repo uses CD to manage deployments to a development environment before PRs are merged.
```

Comment:
Not sure about this one. So far the plans are for CD for commits to `main` which pushes to the production market. Why use CD before merge?

Reply:
CD can be set up to deploy the changes to a development environment for testing before merging the branch to `main` and deploying the changes to production. I agree that we are not there yet, so I will remove this item from the list.